docs/spec.md

```diff
@@ -6268,6 +6294,19 @@ If `x` is a value or placeholder, this function is a shortcut for
 `member_name(type(x))`.  If `x` is not a type that has an appropriate member, or
 a value or a placeholder of such a type, returns `None`.
 
+* `is_convertible_or_quantizable(x: Type, y: Type) -> bool` checks for the
+equality of `x` and `y`, ignoring the bitwidth, when they are of type
+`TensorElementType`. When `x` and `y` are `QuantizedTensorElementType`s,
+the function checks for the equality of `QuantizationExpressedType` component.
+
+```python
+def is_convertible_or_quantizable(x: Type, y: Type) -> Value:
+  return is_integer(x) = is_integer(y) or
+    is_float(x) = is_float(y) or
+    is_complex(x) = is_complex(y) or
+    (is_quantized(x) and is_quantized(y) and expressed_type(x) = expressed_type(y))
```

Comment:
Shouldn't we also have `(is_quantized(x) and expressed_type(type(x))=type(y)) or (is_quantized(y) and expressed_type(type(y))=type(x))`?

Reply:
Per the [RFC](https://github.com/openxla/stablehlo/pull/1664), the criteria for conversion  is limited to `integer -> integer`, `float -> float`, `complex -> complex` and `quantized_type -> quantized_type`. Hence the limited behavior.

Reply:
I just revised the name of the function and add justification to its specific behavior. PTAL.